src/index.html

```diff
@@ -9,7 +9,125 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" autocomplete="off">
+    <fieldset class="form-item">
+      <legend>Personal information</legend>
+      <div class="fieldset-item">
+        <label for="surname">Surname:</label>
+      <input
+      type="text"
+      name="surname"
+      id="surname"
+      required
+      maxlength="10">
+      </div>
```

Comment:
```suggestion
      <div class="fieldset-item">
        <label for="surname">Surname:</label>
        <input
          type="text"
          name="surname"
          id="surname"
          required
          maxlength="10"
        >
      </div>
```

pay attention to your formatting